src/components/estimated-call/DepartureTime.tsx

```diff
@@ -1,22 +1,24 @@
 import {EstimatedCall} from '@atb/api/types/departures';
 import {dictionary, useTranslation} from '@atb/translations';
-import {View} from 'react-native';
+import {ColorValue, View} from 'react-native';
 import {ThemeIcon} from '../theme-icon';
 import {ThemeText} from '../text';
 import {Realtime as RealtimeDark} from '@atb/assets/svg/color/icons/status/dark';
 import {Realtime as RealtimeLight} from '@atb/assets/svg/color/icons/status/light';
-import {StyleSheet, useTheme} from '@atb/theme';
+import {Statuses, StyleSheet, useTheme} from '@atb/theme';
 import {
   formatLocaleTime,
   formatToClockOrRelativeMinutes,
   secondsBetween,
 } from '@atb/utils/date';
 import {EstimatedCallWithMetadata} from '@atb/travel-details-screens/use-departure-data.ts';
+import {ContrastColor, TextColor} from '@atb/theme/colors.ts';
 
 type DepartureTimeProps = {
   departure: EstimatedCall | EstimatedCallWithMetadata;
+  color?: ContrastColor | Statuses | TextColor | ColorValue;
```

Comment:
I think this is a little overkill. Can't it be just `ContrastColor`?

Reply:
Covered in the other PR we did together. I did as I saw in other places.